### DISCUSSION TITLE
Bengali text's weird behavior in neovim

### DISCUSSION THREAD

**Discussion by yishao0925** [ghostty-org/ghostty#3912] [Feature Requests, Ideas] [answered]
1. In normal mode, press $ to go to the end of the text, but the cursor doesn't show at the end of the text.
https://github.com/user-attachments/assets/4f99df79-498d-4d5b-bef9-5780d6b7e179
2.In view mode, select the text and press d to cut, the remain text is not correct.
https://github.com/user-attachments/assets/4ec462d3-7350-40d7-82a6-395dd579dea2

**Comment by UnsaltedScholar** [comment]
I have also seen this issue. It is also apparent across editors aside from neovim (e.g. vim, nano, emacs) and trying several different fonts (monospace, nerd fonts, language-specific).  When bengali text is rendered, all cells before the text render as black (or I am guessing whatever the background color is set as). I cannot say for sure if bengali is the only affected language, but from what I can tell (by perusing default .desktop files which contain samples of various texts) it is the only apparent issue.

**Reply by UnsaltedScholar** [reply]
As of the most recent update, this issue seems to have been resolved.

**Reply by yishao0925** [reply]
The issue still happen for me

**Comment by pluiedev** [comment, marked as answer]
Moved to https://github.com/ghostty-org/ghostty/issues/5637
